src/content/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model.mdx

```diff
@@ -110,18 +110,22 @@ For tips on editing user type, see [Manage user type](#update-user-type).
     id="update-user-type"
     title="Manage user type and user upgrades"
   >
-    Some important tips for managing user type: 
-    * If you're on the [New Relic One pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models), your users' [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) is a billing factor. 
-    * If you change a user's type to a higher type (either via the UI or via an upgrade request email), their user type is changed to that higher user type across all accounts in the organization. 
+Some important tips for managing user type: 
+  * If you're on the [New Relic One pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models), your users' [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) is a billing factor. 
+  * If you change a user's type to a higher type (either via the UI or via an upgrade request email), their user type is changed to that higher user type across all accounts in the organization. 
 
 To update your users' user type:
-    1. For the account you want to manage, go to: **[account dropdown](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#account-dropdown) > Account settings > Users and roles > Users**.
-    2. Either select a user and edit their type or [bulk update the type for multiple users](#bulk).
-    3. If your organization has multiple accounts, you won't be able to see all your organization's users in a single UI location. You'll have to go to the **Users and roles** UI for each account. (Note that the ability to more easily manage your users is one benefit of our [newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models).)
-    
+  1. Go to: **[account dropdown](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#account-dropdown) > Account settings > Users and roles > Users**.
+  2. Either select a user and edit their type or [bulk update the type for multiple users](#bulk).
+  3. Note that if your organization has multiple accounts, you won't be able to see all your users in a single UI location but will have to go to the **Users and roles** UI for each account. 
```

Comment:
How do you feel about moving this note to the end of step 2? I think how you format it here should match how you format in line 124. Alternatively, it could be placed in both cases as its own sentence outside the ordered list.